src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinRTWebSocket.cs

```diff
@@ -154,13 +154,7 @@ private void CancelAllOperations()
 
             if (_webSocketReceiveResultTcs != null)
             {
-                var exception = new WebSocketException(
-                    WebSocketError.InvalidState,
-                    SR.Format(
-                        SR.net_WebSockets_InvalidState_ClosedOrAborted,
-                        "System.Net.WebSockets.InternalClientWebSocket",
-                        "Aborted"));
-
+                var exception = new OperationCanceledException("Aborted");
```

Comment:
We should not be changing this file right now.  This is only built as part of the UWP build of System.Net.WebSockets.Client.  And we have no way of validating this change right now..  We will need to do this change at this point in the future perhaps.